frontends/web/src/locales/en/app.json

```diff
@@ -1013,6 +1013,27 @@
         "title": "Why is there a network fee?"
       }
     },
+    "settings-block-explorer": {
```

Comment:
settingsBlockExplorer, all entries should be camelCase (most are, but there are some misses like right below with settings-electrum :see_no_evil: )